lib/ansible/modules/remote_management/oneview/oneview_ethernet_network.py

```diff
@@ -0,0 +1,249 @@
+    type: dict
+
+ethernet_network_bulk:
+    description: Has the facts about the Ethernet Networks affected by the bulk insert.
+    returned: When 'vlanIdRange' attribute is in data argument. Can be null.
+    type: dict
+
+ethernet_network_connection_template:
+    description: Has the facts about the Ethernet Network Connection Template.
+    returned: On state 'default_bandwidth_reset'. Can be null.
+    type: dict
+'''
+
+from ansible.module_utils.oneview import OneViewModuleBase, OneViewModuleResourceNotFound
+
+
+class EthernetNetworkModule(OneViewModuleBase):
+    MSG_CREATED = 'Ethernet Network created successfully.'
+    MSG_UPDATED = 'Ethernet Network updated successfully.'
+    MSG_DELETED = 'Ethernet Network deleted successfully.'
+    MSG_ALREADY_PRESENT = 'Ethernet Network is already present.'
+    MSG_ALREADY_ABSENT = 'Ethernet Network is already absent.'
+
+    MSG_BULK_CREATED = 'Ethernet Networks created successfully.'
+    MSG_MISSING_BULK_CREATED = 'Some missing Ethernet Networks were created successfully.'
+    MSG_BULK_ALREADY_EXIST = 'The specified Ethernet Networks already exist.'
+    MSG_CONNECTION_TEMPLATE_RESET = 'Ethernet Network connection template was reset to the default.'
+    MSG_ETHERNET_NETWORK_NOT_FOUND = 'Ethernet Network was not found.'
+
+    RESOURCE_FACT_NAME = 'ethernet_network'
+
+    def __init__(self):
+
+        argument_spec = dict(
+            state=dict(
+                required=True,
+                choices=['present', 'absent', 'default_bandwidth_reset']
+            ),
+            data=dict(required=True, type='dict'),
+        )
+
+        super(EthernetNetworkModule, self).__init__(additional_arg_spec=argument_spec, validate_etag_support=True)
+
+        self.resource_client = self.oneview_client.ethernet_networks
+
+    def execute_module(self):
+
+        changed, msg, ansible_facts, resource = False, '', {}, None
+
+        if self.data.get('name'):
+            resource = self.get_by_name(self.data['name'])
+
+        if self.state == 'present':
+            if self.data.get('vlanIdRange'):
+                changed, msg, ansible_facts = self.__bulk_present()
+            else:
+                return self.__present(resource)
+        elif self.state == 'absent':
+            return self.resource_absent(resource)
+        elif self.state == 'default_bandwidth_reset':
+            changed, msg, ansible_facts = self.__default_bandwidth_reset(resource)
+
+        return dict(changed=changed, msg=msg, ansible_facts=ansible_facts)
+
+    def __present(self, resource):
+
+        bandwidth = self.data.pop('bandwidth', None)
+        scope_uris = self.data.pop('scopeUris', None)
+        result = self.resource_present(resource, self.RESOURCE_FACT_NAME)
+
+        if bandwidth:
+            if self.__update_connection_template(result['ansible_facts']['ethernet_network'], bandwidth)[0]:
+                if not result['changed']:
+                    result['changed'] = True
```

Comment:
Good catch! 
I've removed the `if not result['changed']:` from here but left the update to `changed` and `msg`, since `changed` may eq to `False` here if nothing other than the bandwidth of the resource changed.